readme.md

```diff
@@ -24,16 +24,16 @@ plugins {
 }
 ````
 
-Set the container name, and then optionally specify a Dockerfile path, any task
+Set the container name, and then optionally specify a Dockerfile, any task
 dependencies and file resources required for the Docker build. This plugin will
 automatically include outputs of task dependencies in the Docker build context.
 
 **Docker Configuration Parameters**
 - `name` the name to use for this container, may include a tag
 - `tags` (optional) an argument list of tags to create; any tag in `name` will
   be stripped before applying a specific tag; defaults to the empty set
-- `dockerfile` (optional) dockerfile to use for building the image; defaults to
-  `${projectDir}/Dockerfile`
+- `dockerfile` (optional) the dockerfile to use for building the image; defaults to
+  reading the file at the path `${projectDir}/Dockerfile`
```

Comment:
should probably say "defaults to `project.file('Dockerfile')`" and say that this must be a file object.